tox.ini

```diff
@@ -73,18 +73,50 @@ commands_pre =
 commands =
 ; Prefer putting everything in one pylint command to profit from duplication
 ; warnings.
-  black --check --diff .
-  pylint opentelemetry-api/src/opentelemetry \
+  black --check --diff \
```

Comment:
I see that was done to restrict to code and test files in e49a645dde23552f8e03948b3fe0893211507880 but why? Are setup.py files not code, for example?

Reply:
I think it'd be fine to include `setup.py` here, this was just copied from the `pylint` args below.

e49a645 was because tox was failing on my machine, linting a bunch of files in virtualenvs and etc. This seemed like a straightforward improvement to me, even if it means a bit more work keeping the toxfile up to date with the package structure.

Reply:
I had that problem with virtualenvs too but decided that keeping virtualenvs out of the project dir is easier than maintaining theses ugly lists (separation of source and build tree is a nice thing in general).

Reply:
See https://github.com/open-telemetry/opentelemetry-python/pull/174#discussion_r328545664

Reply:
I also vote for not adding it this way. This repo may not add many more directories, but it's quite tricky to add the correct ones and increases the size and error-prone boilerplate considerably.

Maybe we should discuss how we expect to do venvs though? I like using vscode and it's autocompletion doesn't find modules at all with the current structure.